part-4/modularization-concept.md

```diff
@@ -0,0 +1,241 @@
+
+#### Merge behavior "Extend"
+When combining modules `A` and `B` (with the hierarchy `A <- B`), all containers and parameters from module `B` will be added to module `A`. I.e., if module `A` has a container `Organism|Container 1` with two sub-containers `Container 2` and `Container 3` (absolute paths: `Organism|Container 1|Container 2` and `Organism|Container 1|Container 3`), and module `B` has a container `Organism|Container 1` without any sub-containers, the final model will contain `Organism|Container 1` with parameters defined in both modules, and with the sub-containers `Organism|Container 1|Container 2` and `Organism|Container 1|Container 3`.
+
+- **Parameters** are always overwritten. If both modules have a parameter `Organism|Container 1|Param`, the parameter from module `B` will be used. This applies to all properties of the parameter (value, formula, unit, tags etc).
+- **Container types** will be overwritten. If "Organism|Container 1" is "physical" in module "A" and "logical" in module "B", the container will be "logical" in the final model.
+- **Tags** will be extended. If "Organism|Container 1" has a tag "Tag A" in module "A" and a tag "Tag B" in module "B", in the final model, "Organism|Container 1" will have tags "Tag A" and "Tag B".
+- **Neighborhoods** are extended (e.g., addition of new parameters, tags).
+
+### Molecules
+
+Molecules are always overwritten by name. Therefore, if a molecule is defined in an Extension module, it must contain all required parameters. It is not possible to only add some parameters (in merge behavior "Extend"), while retaining parameters from the module that is higher in the hierarchy.
+
+### Reactions
+
+Reactions are always overwritten by name. Therefore, if a reaction is defined in an Extension module, it must contain all required parameters. It is not possible to only add some parameters (in merge behavior "Extend"), while retaining parameters from the module that is higher in the hierarchy.
+
+### Passive transports
+
+#### Merge behavior "Overwrite"
+
+- The equation in the **Kinetic** tab is overwritten.
+
+- The **Parameters** list is overwritten. This also implies that parameters that are not defined in the module that is lower in the hierarchy are removed.
+
+- The **Operator** (and/or) for the "Source" and "Target" lists are overwritten.
+
+- **Source** and **Target** lists are overwritten (i.e., it is possible to remove source/target condition).
+
+- **Include/Exclude** molecule lists for molecules are always extended. However, behavior of the **All checkbox** is overwritten.
+
+The current behavior of combining the passive transports might appear inconsistent and somewhat confusing. The discussion on this topic is still ongoing and can be followed [on GitHub](https://github.com/Open-Systems-Pharmacology/MoBi/issues/2053).
+
+#### Merge behavior "Extend"
+
+Currently, the merge behavior "Extend" is not implemented for passive transports. The behavior is identical to "Overwrite".
```

Comment:
Would change to something like
```
Currently, the merge behavior "Extend" for passive transports is identical to "Overwrite". 
This will change in the future.
```